app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -30,6 +30,7 @@ def self.states
          response_required
          waiting
          invalid_submission
+         stopped
```

Comment:
Can this be reviewed by the team as this is a change to 
states being held in only the state machine yaml ( as stated )

Are we moving away from the consistency of app design? In  my humble opinion we are but may be wrong.

Reply:
@MeenaModhvadia I should have stated in my PR this is the first PR of many! There is a full set of forms/UI for this state, coming soon!